examples/output_voltage_measurement/README.md

```diff
@@ -0,0 +1,44 @@
+## Output Voltage Measurement
+
+This is a MeasurementLink example that sources DC voltage as input with an NI SMU and measures output using NI-VISA DMM.
+
+### Features
+
+- Uses the `nidcpower` and `PyVISA` packages to access NI-DCPower and PyVISA from Python
+- Uses the open-source `PyVISA-sim` package to simulate instruments in software
+- Pin-aware, supporting one session and pin per instrument
+  - Sources the DC voltage level on the selected pin/site
+  - Measures output voltage in a single output pin/site
+- Includes InstrumentStudio and MeasurementLink UI Editor project files
+- For NI-DCPower, uses the NI gRPC Device Server to allow sharing the instrument sessions with other measurement services.
+
+### Required Driver Software
+
+- NI-DCPower
+- NI-488.2 and/or NI-Serial
+- NI-VISA
+- Optional: NI Instrument Simulator software
+
+Note: there is no Python instrument driver for the NI Instrument Simulator, so
+this example directly performs low-level, device-specific commands and queries.
+
+### Required Hardware
+
+By default, this example does not require hardware; it uses a simulated instrument and PyVISA-sim to simulate NI-DCPower and NI-VISA DMM instruments in software. [`NIInstrumentSimulatorV2_0.yaml`](./NIInstrumentSimulatorV2_0.yaml) defines the behavior of the simulated NI-VISA DMM instrument.
+
+This example requires an NI SMU that is supported by NI-DCPower (e.g. PXIe-4141).
+
+To use physical instruments, edit `measurement.py` to specify `USE_SIMULATION = False`
```

Comment:
but in TestStand...

Reply:
Added individual points explaining the edit flow for both MeasurementLink UI Editor / InstrumentStudio and TestStand.